docs/conf.py

```diff
@@ -65,3 +77,18 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 # html_static_path = ['_static']
+
+sphinx_gallery_conf = {
+    # 'filename_pattern': '^((?!skip_).)*$',
```

Comment:
```suggestion
    'filename_pattern': '^((?!skip_).)*$',
```

Reply:
Won't this cause the map.plot method to execute and display the output in RTD?

Reply:
This governs what files to skip for the gallery.

Reply:
We don't use map.plot now, but if used later when implementing other methods like draw_quadrangle, this file is executed and the map is plotted (not sure if we need that). This file is only parsed and not executed.

Reply:
I am confused, what does this have to do with the map.plot method?

Reply:
Sorry I got confused, there was an issue with one of the draw methods on the local installation of sunpy on my system.

Reply:
Is it a bug or just a setup issue on your machine?

Reply:
I had written code for draw_quadrangle which required map.plot() to be called first. So I assumed that we would need to call map.plot which would basically end up plotting the map in 2D which we wouldn't need for this project.